infrastructure/main.tf

```diff
@@ -22,6 +22,10 @@ locals {
   nonPreviewVaultName    = "${var.product}-ft-api-${var.env}"
   vaultName              = "${(var.env == "preview" || var.env == "spreview") ? local.previewVaultName : local.nonPreviewVaultName}"
 
+  cmcPreviewVaultName = "cmc-aat"
+  cmcNonPreviewVaultName = "cmc-${var.env}"
+  cmcVaultName = "${(var.env == "preview" || var.env == "spreview") ? local.previewVaultName : local.nonPreviewVaultName}"
```

Comment:
same as `vaultName` - we can re-use that

Reply:
oops, was a typo